packages/core/src/components/Table/components/TableSortingComponent.tsx

```diff
@@ -0,0 +1,82 @@
+import { useState } from 'react';
```

Comment:
We don't use the nested `/components` directory in the project. Consider creating a sub-directory name by the component, e.g. `TableSorting`, or just put the component into the `Table/` directory.

Reply:
If it's for Stories only, them move into the `./stories` directory as we usually do.